src/kd_time_main.cpp

```diff
@@ -0,0 +1,28 @@
+#include "polyhedralGravity/input/YAMLConfigReader.h"
+#include <benchmark/benchmark.h>
+
+namespace polyhedralGravity {
+
+    Polyhedron createBigPolyhedron(const PolyhedralSource &source, const PlaneSelectionAlgorithm::Algorithm &algorithm) {
+        return Polyhedron(source, 1.0, NormalOrientation::OUTWARDS, PolyhedronIntegrity::HEAL, algorithm);
+    }
+
+    void BM_Polyhedron_Tree(benchmark::State &state, const PlaneSelectionAlgorithm::Algorithm &algorithm) {
+        const std::vector<std::string> configPaths{"../example-config/example_eros.yaml", "../example-config/example_eros_scaled_1.yaml", "../example-config/example_eros_scaled_2.yaml", "../example-config/example_eros_scaled_3.yaml", "../example-config/example_eros_scaled_4.yaml"};
+        const std::shared_ptr<ConfigSource> config = std::make_shared<YAMLConfigReader>(configPaths.at(state.range(0)));
+        const PolyhedralSource source = config->getDataSource()->getPolyhedralSource();
```

Comment:
You don't require a full config here. Just use the `TetGenAdapter` and hand over the input file. No dedicated config file required.